juliet/README.md

```diff
@@ -0,0 +1,23 @@
+# `juliet` protocol implementation
+
+This crate implements the juliet multiplexing protocol as laid out in the [Juliet RFC](https://github.com/marc-casperlabs/juliet-rfc/blob/master/juliet.md). It aims to be a secure, simple, easy to verify/review implementation that is still reasonably performant.
+
+## Benefits
+
+ The Juliet protocol comes with a core set of features, such as
+
+* carefully designed with security and DoS resilience as its foremoast goal,
+* customizable frame sizes,
+* up to 256 multiplexed, interleaved channels,
+* backpressure support fully baked in, and
+* low overhead (4 bytes per frame + 1-5 bytes depending on payload length).
+
+This crate's implementation includes benefits such as
+
+* a side-effect free implementation of the Juliet protocol,
+* an `async` IO layer integrated with the [`bytes`](https://docs.rs/bytes) crate to use it, and
+* a type-safe RPC layer built on top.
+
+## Examples
+
+For a quick usage example, see `examples/fizzbuzz.rz`.
```

Comment:
I had that originally, but it leads to a dead link in the rustdocs. Are you sure it's preferable?

Reply:
I'm typo-blind, my biggest weakness!

Strongly against duplicating the text, especially since it is bound to change some down the line and I'd hate for that to get out of sync.

The rustdoc links will always be dead, since it can be moved around and docs.rs RS won't have the examples uploaded, I think. The alternative would be linking to an example on the web (i.e. the GitHub repo), which will cause a version mismatch when someone views older docs.

Which one is better?

Reply:
Typo fixed in b2c6855b3